src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/FileLength/newline/expected.txt

```diff
@@ -1,3 +1,4 @@
 Starting audit...
 [ERROR] /Users/hgh/Downloads/my_file/IDEA_workspace/patchfiltergeneratepatch/src/main/java/Checker/FileLength/Test2.java:1: File length is 14 lines (max allowed is 5). [FileLength]
+[ERROR] /Users/hgh/Downloads/my_file/IDEA_workspace/patchfiltergeneratepatch/src/main/java/Checker/FileLength/Test.java:1: File length is 13 lines (max allowed is 5). [FileLength]
```

Comment:
How these and below changes are related to the main problem your are fixing?

Reply:
> How these and below changes are related to the main problem your are fixing?

@rdiachenko same reason in https://github.com/checkstyle/patch-filters/pull/96#discussion_r456180795, when I remove original `@Ignore(until https://github.com/checkstyle/patch-filters/issues/88)`,  found these checks still need to be solved by context strategy, and the reason of this change is  original `expected.txt` is not correct, I did not find it until remove `@Ignore(until https://github.com/checkstyle/patch-filters/issues/88)`